src/Journey/Interaction.py

```diff
@@ -0,0 +1,88 @@
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from Journey.Play import Play
+
+class UnknownAnswer(Exception):
+    def __init__(self):
+        super().__init__("Unknown answer")
+
+class Interaction:
```

Comment:
Well, it can easily be extended to handle that as well. However, I asked about this not too long ago and the agreement was to provide answer possibilities.

Reply:
That is certainly true for combat and general exploration, but I don't think we agreed on that for dialogue? I hope I didn't misunderstand that, because I'm not sure if I _can_ get that working with acceptable quality and latency.

Reply:
I've changed it a little bit, so now it can accept user typed answers too.